src/NHSD.BuyingCatalogue.Ordering.Contracts/IOrderService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```

Comment:
I don't think we need these `using` statements.

Reply:
Removed `using `statements
```
using System.Collections.Generic;
using System.Linq;
using System.Text;
```